lib/active_job/queue_adapters/sidekiq_adapter.rb

```diff
@@ -17,7 +17,7 @@ def perform(job_data)
     end
   end
 
-  unless ActiveJob::Base.respond_to?(:sidekiq_options)
+  ActiveSupport.on_load(:active_job) do
```

Comment:
This will trigger only if railties exist. 

What if i ant a script with AJ and Sidekiq only ?

Reply:
I'm pretty sure this works without Railties - it's part of ActiveSupport:

```ruby
require "bundler/inline"
gemfile do
  gem "activejob"
end

require "active_job"

ActiveSupport.on_load :active_job do
  puts "In load hook!"
end

puts "Loading ActiveJob::Base..."
ActiveJob::Base
puts "Done!"
```

prints

```
Loading ActiveJob::Base...
In load hook!
Done!
```